### DISCUSSION TITLE
For v3 model, is there a pretrained model with ResNet 18 backbone?

### DISCUSSION THREAD

**Discussion by gliufetch** [PaddlePaddle/PaddleOCR#12373] [Q&A] [answered]
for v2 detection, there is a pretrained model with [res18 backbone, ch_ppocr_server_v2.0_det](https://github.com/PaddlePaddle/PaddleOCR/blob/0525f6bb01bfed401f767894619f6a25ee750892/doc/doc_ch/models_list.md#:~:text=ch_ppocr_server_v2.0_det).

is there a pretrained one with res18 backbone for v3 detection?
Thank you!

**Comment by tink2123** [comment, marked as answer]
Sorry, v3 does not provide a model for large backbone
